src/mapping.ts

```diff
@@ -1,8 +1,14 @@
-import { dataSource } from '@graphprotocol/graph-ts';
-import { DelegateChanged, DelegateVotesChanged } from '../generated/Token/Token';
+import { Address, dataSource } from '@graphprotocol/graph-ts';
+import { DelegateChanged, DelegateVotesChanged } from '../generated/templates/GenericERC20Votes/Token';
 import { toDecimal, getDelegate, getGovernance, BIGINT_ZERO } from './helpers';
+import { ContractDeployed } from "../generated/GeneralPurposeFactory/GeneralPurposeFactory";
+import { GenericERC20Votes } from "../generated/templates";
+import { log } from '@graphprotocol/graph-ts';
```

Comment:
This pkg is already imported earlier but anyhow you should remove the log from the code.
```suggestion
```

Reply:
Ok sorry, thought I'd mimic what we have on [delegates-api](https://github.com/snapshot-labs/delegates-api/blob/d9059145960769c6da502b25412119059c15a63a/src/writer.ts#L10)